pixeltable/iterators/document.py

```diff
@@ -79,9 +93,13 @@ def __init__(
         if self._doc_handle.format == DocumentType.DocumentFormat.HTML:
             assert self._doc_handle.bs_doc is not None
             self._sections = self._html_sections()
-        else:
+        elif self._doc_handle.format == DocumentType.DocumentFormat.MD:
             assert self._doc_handle.md_ast is not None
             self._sections = self._markdown_sections()
+        else:
```

Comment:
Add a comment indicating that this implies `self._doc_handle.format == DocumentType.DocumentFormat.PDF`